### PR TITLE
Fix --watch option with node on Mac OS

### LINKED ISSUE #399: Dart Sass --watch fails on the command line
**Expected Behavior**
Given I'm using dart sass on the command line inside of an npm script
When I run sass --watch sass/index.scss dist/index.css and then change my scss file
Then the sass should compile to css once after npm script and again after my change

**Actual Behavior**
Given I'm using dart sass on the command line inside of an npm script
When I run sass --watch sass/index.scss dist/index.css and then change my scss file
Then the sass compiles to css once after npm script but after my change I receive the following error

```
project_name/node_modules/sass/sass.dart.js:6421
throw H.e(P.lF(this,b.gnt(),b.gnL(),b.gnx(),null))},null,"gny",5,0,null,19],
^

NoSuchMethodError: method not found: 'call'
Receiver: Closure 'Hs'
Arguments: ["sass/index.scss"]
    at Object.e (project_name/node_modules/sass/sass.dart.js:1642:3)
    at Hs.dart.k.kj (project_name/node_modules/sass/sass.dart.js:6421:9)
    at Object.J.pj (project_name/node_modules/sass/sass.dart.js:22083:34)
    at Object.lQ (project_name/node_modules/sass/sass.dart.js:1605:10)
    at Object.tG (project_name/node_modules/sass/sass.dart.js:1616:21)
    at Object.tH (project_name/node_modules/sass/sass.dart.js:1611:10)
    at Object.fx (project_name/node_modules/sass/sass.dart.js:5706:28)
    at dart.Cb (project_name/node_modules/sass/sass.dart.js:7294:10)
    at FSWatcher.<anonymous> (project_name/node_modules/sass/sass.dart.js:7282:42)
    at emitOne (events.js:116:13)
```

**Steps to Reproduce**
1) run `npm install --save sass` in your command line
2) create some sass in sass/index.scss
3) add `sass --watch sass/index.scss dist/index.css` to your `npm start` script
3) run `npm start`
4) make a change to your sass file and save your change.

**Additional Information**
System: MacOS 10.13.5
dart-sass version: 1.9.0
node version: 8.11.1
npm version: 6.1.0

### PR DESCRIPTION
**Pull request by jmesserly**
Closes #399

### CODE REVIEW

**Comment by jmesserly** [comment]
It appears that only one argument is passed to the JS callback on Mac OS+node for `'add'` and `'change'` events. dart2js treats this as a noSuchMethodError (which I think is wrong, fwiw: [Dart SDK issue 33134](https://github.com/dart-lang/sdk/issues/33134#issuecomment-389349558)).

I'm not sure if we have Mac buildbot coverage? Let me know if I should try to add a test of some sort.

**Comment by nex3** [comment]
Unfortunately, we don't have Mac coverage. Travis offers Mac OS bots, but they don't have many machines which makes the iteration times unbearably slow :confused:.

**Comment by jmesserly** [comment]
yeah ... if the tradeoff is fixing an easy thing like this every now and then, seems okay :). (IMO this really is a dart2js bug anyways)

**Comment by jmesserly** [comment]
oh also: do you typically do changelog updates for this kind of change?

**Comment by nex3** [comment]
Yeah... can you bump the pubspec version to 1.9.1-dev and add a changelog entry mentioning that a `--watch` crash on Mac OS was fixed?

**Comment by jmesserly** [comment]
I added a changelog entry if you want to take a look

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.9.2-dev
```
Comment (nex3) [inline CHANGELOG.md]: Nit: We usually don't add `-dev` to changelog entries; putting that in the pubspec is sufficient to ensure that users don't think this is a full release version.

**Comment by Amaruq** [comment]
Works fine! Thank you!

**Pull request merged**